### PR TITLE
[T2A4][T16-A2]ChingHuiQi

### PR DESCRIPTION
**Pull request by HQching**
Got help on writing the comparator syntax for the sorting function from http://stackoverflow.com/questions/4699807/sort-arraylist-of-array-in-java

### CODE REVIEW

**Comment by HQching** [comment]
don't know how to do the rest...Ready for review

**Comment by HQching** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -467,32 +479,53 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
...
-                matchedPersons.add(person);
-            }
+            findMatchCaseInsensitive(keywords, matchedPersons, person, wordsInName);
+
+      
         }
         return matchedPersons;
     }
 
+    private static void findMatchCaseInsensitive(Collection<String> keywords, final ArrayList<String[]> matchedPersons,
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java]: notice the arrow head like code indentation; it is one of the indications that SLAP may not be so good in the code.
try to improve it!

File: README.md [hunk 2]
```diff
@@ -79,11 +79,17 @@ oldest first.
 
 Format: `list`  
 
+#### Listing all persons in alphabetical order: `sort`
+
+> Shows a list of persons, as an indexed list, in increasing alphabetical order.
```
Comment (okkhoy) [inline README.md:84]: good to see you have updated the documentation :+1:

File: src/seedu/addressbook/AddressBookWithEditCommand.java [hunk 3]
```diff
@@ -0,0 +1,1319 @@
...
+        return String.format(MESSAGE_DELETE_PERSON_SUCCESS, getMessageForFormattedPersonData(deletedPerson));
+    }
+
+    /**
+     * Edits person identified using last displayed index.
+     *
+     * @param commandArgs full command args string from the user
+     * @return feedback display message for the operation result
+     */
+    private static String executeEditPerson(String commandArgs) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBookWithEditCommand.java:622]: this method seems to be doing a lot of things? try to improve SLAP on this one

**Review by okkhoy — COMMENTED** [review]
the advantage of using RCS is that you can branch out as much as you want to write any kind of code, you can recover the previous version from your commits. as such no need to add a new file for a functionality.

**Comment by okkhoy** [comment]
@HQching 
you need to update the tests as well when you add new functionality.

some comments added. please close the PR after you have noted them

**Pull request closed without merging**